src/nbl/asset/interchange/CPLYMeshFileLoader.h

```diff
@@ -54,6 +54,37 @@ class CPLYMeshFileLoader : public IRenderpassIndependentPipelineLoader
     virtual SAssetBundle loadAsset(io::IReadFile* _file, const IAssetLoader::SAssetLoadParams& _params, IAssetLoader::IAssetLoaderOverride* _override = nullptr, uint32_t _hierarchyLevel = 0u) override;
 private:
 
+	enum E_TYPE { ET_POS = 0, ET_UV = 2, ET_NORM = 3, ET_COL = 1 };
+
+	void initialize();
```

Comment:
use the `override` keyword always when you're overriding a virtual function

Reply:
I had no clue it's virtual

Reply:
corrected in following commit